cfme/services/catalogs/catalog_item.py

```diff
@@ -225,8 +225,8 @@ def create(self):
         sel.force_navigate('catalog_item_new',
                            context={'provider_type': self.item_type})
         sel.wait_for_element(basic_info_form.name_text)
-        catalog = fakeobject_or_object(self.catalog, "name", "<Unassigned>")
-        dialog = fakeobject_or_object(self.dialog, "name", "<No Dialog>")
+        catalog = fakeobject_or_object(self.catalog, "name", "Unassigned")
```

Comment:
Not sure this is the same for both 5.5 and 5.6, but we can wait and see what the result is on PRT